change/@fluentui-react-datepicker-compat-59a888f8-093f-449c-b49e-4f580894da5e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 15 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27637&commitId=b084e0bf3ef84496de9829ec9c5737d4b93dec1e&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1916 screenshots unchanged, 0 screenshots with different dimensions and 15 screenshots with visible difference.


  <details><summary><b>unknown</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|5447| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|10395| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|12660| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|6243| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|5944| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|5776| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|4215| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|7976| Changed|
|DatePicker Compat.default.opened.chromium.png|5447| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|4239| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|7954| Changed|
|DatePicker Compat.marked dates.default.chromium.png|5451| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|4409| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|8603| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|5711| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>